Copyright (C) 2017 XLAB, Ltd.

This work is open source software, licensed under the terms of the
BSD license as described in the LICENSE file in the top-level directory.

diff --git a/mike-apps/OpenFOAM/GET b/mike-apps/OpenFOAM/GET
index 72ab5c2..e8cad05 100755
--- a/mike-apps/OpenFOAM/GET
+++ b/mike-apps/OpenFOAM/GET
@@ -109,7 +109,6 @@ install $SRCDIR/platforms/$WM_OPTIONS/lib/libforces.so $ROOTFS/usr/lib
 cp -r $SRCDIR/etc $ROOTFS/openfoam

 echo "
-/usr/bin/**: ${ROOTFS}/usr/bin/**
 /usr/lib/**: ${ROOTFS}/usr/lib/**
 /openfoam/etc/**: $ROOTFS/openfoam/etc/**
 " > usr.manifest
